lib/srv/discovery/access_graph_aws.go

```diff
@@ -530,7 +530,7 @@ func (s *Server) initTAGAWSWatchers(ctx context.Context, cfg *Config) error {
 				// We will wait for the config to change and re-evaluate the fetchers
 				// before starting the sync.
 				if len(allFetchers) == 0 {
-					s.Log.DebugContext(ctx, "No AWS sync fetchers configured. Access graph sync will not be enabled.")
+					s.Log.DebugContext(ctx, "No AWS sync fetchers with trail enabled configured. Access graph sync will not be enabled.")
```

Comment:
What does "with trail enabled" mean?

Reply:
It's the same error message as below, so I wanted to differentiate it to know where it comes from. I think it means CloudTrail fetching. Updated.

Reply:
ok, I have updated them to say:

`Access graph sync with CloudTrail will will not be enabled.` and
`Access graph sync without CloudTrail will will not be enabled.`

It's a little cheeky, but the main thing for me is to find the right place for error messages. I don't really understand the full details of these fetchers either.